fdb-record-layer-lucene/src/test/java/com/apple/foundationdb/record/lucene/highlight/LuceneScaleTest.java

```diff
@@ -159,6 +162,18 @@ private static class Config {
          * beginning;, this can be any string.
          */
         static final String ISOLATION_ID = "default";
+
+        /**
+         * The probability (in promills) of performing an explicit merge, if merge is requested.
```

Comment:
Is `promills` a typo, or an abbreviation I'm unfamiliar with?

Reply:
Typo -  permille.

Reply:
Reworded. 
✅